src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -1725,6 +1796,639 @@ public static void assertValidStrand( final Strand strand ) {
+        b37ToHg19ContigNameMap.put( "GL000215.1", "GL000215.1" );
+        b37ToHg19ContigNameMap.put( "GL000225.1", "GL000225.1" );
+        b37ToHg19ContigNameMap.put( "GL000216.1", "GL000216.1" );
+        b37ToHg19ContigNameMap.put( "GL000194.1", "GL000194.1" );
+        b37ToHg19ContigNameMap.put( "GL000217.1", "GL000217.1" );
+        b37ToHg19ContigNameMap.put( "GL000197.1", "GL000197.1" );
+        b37ToHg19ContigNameMap.put( "GL000222.1", "GL000222.1" );
+        b37ToHg19ContigNameMap.put( "GL000200.1", "GL000200.1" );
+        b37ToHg19ContigNameMap.put( "GL000211.1", "GL000211.1" );
+        b37ToHg19ContigNameMap.put( "GL000247.1", "GL000247.1" );
+        b37ToHg19ContigNameMap.put( "GL000233.1", "GL000233.1" );
+        b37ToHg19ContigNameMap.put( "GL000210.1", "GL000210.1" );
+        b37ToHg19ContigNameMap.put( "GL000198.1", "GL000198.1" );
+        b37ToHg19ContigNameMap.put( "GL000245.1", "GL000245.1" );
+        b37ToHg19ContigNameMap.put( "GL000234.1", "GL000234.1" );
+        b37ToHg19ContigNameMap.put( "GL000203.1", "GL000203.1" );
+        b37ToHg19ContigNameMap.put( "GL000239.1", "GL000239.1" );
+        b37ToHg19ContigNameMap.put( "GL000213.1", "GL000213.1" );
+        b37ToHg19ContigNameMap.put( "GL000227.1", "GL000227.1" );
+        b37ToHg19ContigNameMap.put( "GL000208.1", "GL000208.1" );
+        b37ToHg19ContigNameMap.put( "GL000230.1", "GL000230.1" );
+        b37ToHg19ContigNameMap.put( "GL000231.1", "GL000231.1" );
+        b37ToHg19ContigNameMap.put( "GL000228.1", "GL000228.1" );
+        b37ToHg19ContigNameMap.put( "GL000243.1", "GL000243.1" );
+        b37ToHg19ContigNameMap.put( "GL000229.1", "GL000229.1" );
+        b37ToHg19ContigNameMap.put( "GL000205.1", "GL000205.1" );
+        b37ToHg19ContigNameMap.put( "GL000224.1", "GL000224.1" );
+        b37ToHg19ContigNameMap.put( "GL000191.1", "GL000191.1" );
+        b37ToHg19ContigNameMap.put( "GL000196.1", "GL000196.1" );
+        b37ToHg19ContigNameMap.put( "GL000193.1", "GL000193.1" );
+        b37ToHg19ContigNameMap.put( "GL000201.1", "GL000201.1" );
+        b37ToHg19ContigNameMap.put( "GL000237.1", "GL000237.1" );
+        b37ToHg19ContigNameMap.put( "GL000246.1", "GL000246.1" );
+        b37ToHg19ContigNameMap.put( "GL000241.1", "GL000241.1" );
+        b37ToHg19ContigNameMap.put( "GL000204.1", "GL000204.1" );
+        b37ToHg19ContigNameMap.put( "GL000207.1", "GL000207.1" );
+        b37ToHg19ContigNameMap.put( "GL000209.1", "GL000209.1" );
+        b37ToHg19ContigNameMap.put( "GL000219.1", "GL000219.1" );
+        b37ToHg19ContigNameMap.put( "GL000220.1", "GL000220.1" );
+        b37ToHg19ContigNameMap.put( "GL000236.1", "GL000236.1" );
+
+        return b37ToHg19ContigNameMap;
+    }
+
+    /**
+     * @return An initialized {@link SAMSequenceDictionary} containing the {@link SAMSequenceRecord}s from the B37 Human Genome Reference.
+     */
+    private static final SAMSequenceDictionary initializeB37SequenceDict() {
```

Comment:
Is there really no other way to do this?  This seems like a lot of hardcoding for what could just be a resource file in the jar and then read in with one line of code.

Reply:
That's true.  I didn't think about including the sequence dictionary in the jar.  

I'll fix it.